README.md

```diff
@@ -1,109 +1,100 @@
-# content-editorial-read
-_Should be the same as the github repo name but it isn't always._
+# Draft content API
 
-[![Circle CI](https://circleci.com/gh/Financial-Times/content-editorial-read/tree/master.png?style=shield)](https://circleci.com/gh/Financial-Times/content-editorial-read/tree/master)[![Go Report Card](https://goreportcard.com/badge/github.com/Financial-Times/content-editorial-read)](https://goreportcard.com/report/github.com/Financial-Times/content-editorial-read) [![Coverage Status](https://coveralls.io/repos/github/Financial-Times/content-editorial-read/badge.svg)](https://coveralls.io/github/Financial-Times/content-editorial-read)
+[![Circle CI](https://circleci.com/gh/Financial-Times/draft-annotations-api/tree/master.png?style=shield)](https://circleci.com/gh/Financial-Times/draft-annotations-api/tree/master)[![Go Report Card](https://goreportcard.com/badge/github.com/Financial-Times/draft-annotations-api)](https://goreportcard.com/report/github.com/Financial-Times/draft-annotations-api) [![Coverage Status](https://coveralls.io/repos/github/Financial-Times/draft-annotations-api/badge.svg)](https://coveralls.io/github/Financial-Times/draft-annotations-api)
 
 ## Introduction
 
-_What is this service and what is it for? What other services does it depend on_
-
-UPP Golang Microservice Template short description - please amend
+Draft Annotations API is a microservice that provides access to draft annotations for content stored in PAC. Currently, the service is a simple proxy to UPP Public Annoations API.
 
 ## Installation
-      
-_How can I install it_
 
 Download the source code, dependencies and test dependencies:
 
-        go get -u github.com/kardianos/govendor
-        go get -u github.com/Financial-Times/content-editorial-read
-        cd $GOPATH/src/github.com/Financial-Times/content-editorial-read
-        govendor sync
-        go build .
+```
+go get -u github.com/kardianos/govendor
+go get -u github.com/Financial-Times/draft-annotations-api
```

Comment:
Actually this is probably _not_ what you want to do. At this point do a standard `git clone`, then use `govendor sync` as below to pull in the dependencies.